compiler.binarybuilder.bb.fc

```diff
@@ -0,0 +1,49 @@
+# Fortran compiler for BinaryBuilder
+
+FORTRAN='$FC'
+BASIC='-c -fno-second-underscore -fPIC'
+MBASIC='-fno-second-underscore -fPIC'
+LIBCMD=''
+MODCMD='-I$MOD'
+MODCMDB=$MODCMD
+MVMODS='$(MV) -f $(OBJ)/*.mod $(MOD)/ || true'
+OPTIMIZATION='-O2'
+NOOPTIMIZATION='-O0'
+DEBUG=
+OPENMP='-fopenmp'
+SOMP='-fopenmp'
+F77='-ffixed-form'
+F90=''
+F95=''
+NOFMAIN=''
+CCONDEF=
+USUAL=
+SPECIAL=
+F77SUFFIX=f90
+F95SUFFIX=f90
+CUDA=
+CUDACOMP=$FORTRAN
+CUDAARCH=
+CUDACODE=
+CUDABASIC=
+CUDALIBS=
+CUDAOPENMP=$OPENMP
+MOMP='-lgomp'
+TIMER=GEN
+BLAS='-lopenblas'
+LAPACK='-lopenblas'
```

Comment:
could these be LBT?

Reply:
Yes, LBT is just not working on Windows for Julia < v"1.9".
But GALAHAD is probably not working on Windows.